aya-log/src/lib.rs

```diff
@@ -73,6 +73,17 @@ use aya::{
     Bpf, Pod,
 };
 
+#[derive(Copy, Clone)]
+struct RecordField(aya_log_common::RecordField);
```

Comment:
Also, wouldn't it better to name these RecordFieldWrapper ArgumentWrapper and DisplayHintWrapper so that we don't have the name clash and don't have to use the aya_log_common:: prefix everywhere else?

Reply:
Done, thanks